Public/Src/App/Bxl/Args.cs

```diff
@@ -1018,6 +1018,9 @@ public bool TryParse(string[] args, PathTable pathTable, out ICommandLineConfigu
                         OptionHandlerFactory.CreateBoolOption(
                             "useHardlinks",
                             sign => engineConfiguration.UseHardlinks = sign),
+                        OptionHandlerFactory.CreateBoolOption(
+                            "useHistoricalCpuUsageInfo",

```

Comment:
>useHistoricalCpuUsageInfo [](start = 29, length = 25)

Now that this is dynamic, it might be good to include it in the execution log for ease of postmortem datamining. Food for thought, no need to do that in this PR